hoverctl/wrapper/hoverfly.go

```diff
@@ -188,8 +189,45 @@ func SetMiddleware(target Target, binary, script, remote string) (v2.MiddlewareV
 	return apiResponse, nil
 }
 
+func GetLogs(target Target, format string) ([]string, error) {
+	headers := map[string]string{
+		"Content-Type": "text/plain",
+	}
+
+	if format == "json" {
+		headers["Content-Type"] = "application/json"
+	}
+
+	response, err := doRequest(target, "GET", v2ApiLogs, "", headers)
+	if err != nil {
+		return []string{}, err
+	}
+
+	defer response.Body.Close()
+
+	responseBody, _ := ioutil.ReadAll(response.Body)
```

Comment:
Ideally, this would happen externally. So you can use the `Accept` header to say whether you want JSON or text back from the endpoint. `Accept: application/json` means it will return JSON `Accept: text/*` text. Then you can just move this logic to the endpoint